Tests/FluentAssertions.Json.Shared.Specs/JTokenAssertionsSpecs.cs

```diff
@@ -662,6 +662,56 @@ public void When_jtoken_does_have_element_NotHaveElement_should_fail()
 
         #endregion (Not)HaveElement
 
+        #region HaveElementAndValue
+
+        [Fact]
+        public void When_jtoken_has_element_and_value_HaveElementAndValue_should_succeed()
```

Comment:
We prefer not to use the exact name of a code element in the test name. I would rephrase this as `When_json_has_the_expected_element_and_value_it_should_succeed`